src/pages/en/reference/cli-reference.md

```diff
@@ -24,13 +24,17 @@ Sets which network IP addresses the dev server should listen on (i.e. non-localh
 
 ### `astro build`
 
-Builds your site for production.
+Builds your site for deployment. By default, this will generate static files in to a `dist/` directory. If [SSR is enabled](/en/guides/server-side-rendering/), this will generate the necessary server files to serve your site. 
```

Comment:
```suggestion
Builds your site for deployment. By default, this will generate static files and place them in a `dist/` directory. If [SSR is enabled](/en/guides/server-side-rendering/), this will generate the necessary server files to serve your site. 
```